src/bindings/python/nixl-meta/nixl_meta_utils.py

```
@@ -0,0 +1,97 @@
+        return None
+
+    version_py = pathlib.Path(spec.origin).parent / "version.py"
+    try:
+        version_spec = importlib.util.spec_from_file_location(
+            "_nixl_torch_version", version_py
+        )
+        if version_spec is None or version_spec.loader is None:
+            return None
+        module = importlib.util.module_from_spec(version_spec)
+        version_spec.loader.exec_module(module)
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)